code/modules/research/bepis.dm

```diff
@@ -0,0 +1,225 @@
+//This system is designed to act as an in-between for cargo and science, and the first major money sink in the game outside of just buying things from cargo (As of 10/9/19, anyway).
+
+//economics defined values, subject to change should anything be too high or low in practice.
+
+
+/obj/machinery/rnd/bepis
+	name = "\improper B.E.P.I.S. Chamber"
+	desc = "A high fidelity testing device which unlocks the secrets of the known universe using the two most powerful substances available to man: excessive amounts of electricity and capital."
+	icon = 'icons/obj/machines/bepis.dmi'
+	icon_state = "chamber"
+	layer = ABOVE_MOB_LAYER
```

Comment:
This causes it to show above items. Is this for the deer? Dont do this for the deer, and leave it at machines default.

Reply:
The problem is that otherwise the player icon sits on-top of the tube itself, since it's a 2 tile tall sprite. I do wish there was a better in-between than this though.
![image](https://user-images.githubusercontent.com/41715314/71340585-47646180-2525-11ea-9937-4a2dc9090dcf.png)

Reply:
Ah, cloner got the same issue doesnt it. Hmm